### PR TITLE
Scaffolds basic wazero.io

### PR DESCRIPTION
**Pull request by codefromthecrypt**
Thanks to @peterj for the logo. This adds a site re-using the same approach of func-e (mostly to save time!). We can add more content and move stuff from the README over after the basics are up.

Test locally (with go 1.18) via `make site`

take a look here https://deploy-preview-538--wazero.netlify.app/

### CODE REVIEW

**Comment by codefromthecrypt** [comment]
marking draft as the deploy preview sees nada for some reason https://627a55d1b33cde0008cb27bb--wazero.netlify.app/

File: site/content/_index.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+means you can run code written in other languages that compile to WebAssembly
+modules.
+
+## Example
+
+The best way to learn wazero is by trying one of our [examples][1]
+
+For the impatient, here's how invoking a factorial function looks in wazero:
+
+```golang
```
Comment (mathetake) [inline site/content/_index.md:20]: syntax highlighting seems not working correctly on the site :(

Comment (codefromthecrypt) [reply site/content/_index.md]: fixed.. it likes "go" not "golang"

**Review by mathetake — APPROVED** [review]

**Comment by peterj** [comment]
LGTM

**Review by peterj — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -88,3 +90,8 @@ check:
 		echo "The following differences will fail CI until committed:"; \
 		git diff --exit-code; \
 	fi
+
+.PHONY: site
+site: ## Serve website content
+	@git submodule update
```
Comment (anuraaga) [inline Makefile:96]: Does it work to add `--init` here? Or otherwise the user will have to run that manually once?

**Review by anuraaga — APPROVED** [review]

File: Makefile (hunk 2 above)
Comment (codefromthecrypt) [reply Makefile:96]: will add sure

**Pull request merged**